contracts/JBController.sol

```diff
@@ -3,12 +3,12 @@ pragma solidity ^0.8.16;
 
 import {ERC165} from '@openzeppelin/contracts/utils/introspection/ERC165.sol';
```

Comment:
what's the verdict for `JBController` in this PR? we A) keeping it _and_ 3.2? B) folding everything into it? or C) folding everything into 3.2?